### ISSUE TITLE
something wrong with the loss

### ISSUE THREAD

**Issue by tanjiarui** [AlexanderBogatko/Keras-DeTR#1]
Hi there, looks like there are some errors in the loss class
I think it should be `predicted_boxes[batch_id]` in line [65](https://github.com/AlexanderBogatko/Keras-DeTR/blob/69f6309c3b377257beba705819d19f5ff650a92a/losses/detr_loss.py#L65)
There is a typo in line [97](https://github.com/AlexanderBogatko/Keras-DeTR/blob/69f6309c3b377257beba705819d19f5ff650a92a/losses/detr_loss.py#L97). It should be `self.regression_loss`
Also, there is no way to run the fit cause ur loss function didn't follow the tf standard and mixed with some non-tensors. Setting `run_eagerly=True` can run it, bit it's not preferable

**Comment by AlexanderBogatko** [comment]
Hi! Thanks for review! I fixed bugs in lines 65 and 97, about eagerly mode: you can try to use `tf.py_function` to perform hungarian matching or implement it in pure TF, unfortunately, i don't have enough time now to do that.
